hyperapp.d.ts

```diff
@@ -0,0 +1,68 @@
+export as namespace hyperapp
+
+/** @namespace [Virtual DOM] */
+
+/** The Virtual DOM representation of an Element
+ *
+ * @memberOf [Virtual DOM]
+*/
+export interface VirtualNode<Data> {
+  tag: string
+  data: Data
+  children: VirtualNodeChild<{} | null>[]
```

Comment:
In strict mode `null` need to be ensured, this is prerequisite if you are using : `h('tag', null)` node